quiz/backend/players/tests/test_views.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import status
+
+from players.views import NicknameGeneratorAPIView
+
+
+def test_nickname_generator_view_returns_correct_data(api_rf):
+    request = api_rf.get("/players/nickname-generator/")
```

Comment:
Yup, you changed the path, and forgot to change it in test.😅

Reply:
done